src/tokens/tron.ts

```diff
@@ -0,0 +1,13 @@
+import { ChainId } from '../chainIds';
+import { IToken } from '../interfaces';
+
+export const tokens: IToken[] = [
+  {
+    name: 'TRX',
+    address: '0x0000000000000000000000000000000000000000',
```

Comment:
change zero address `T9yD14Nj9j7xAB4dbGeiX9h8unkKHxuWwb`

ask to @Ignacio-87 what tokens we'll need to add in mainnet

Reply:
Done!